libs/libbuiltin/libgcc/Make.defs

```diff
@@ -19,10 +19,12 @@
 ############################################################################
 
 ifeq ($(CONFIG_PROFILE_MINI),y)
-
 CSRCS += profile.c
+endif
+
+ifeq ($(CONFIG_COVERAGE_MINI)$(CONFIG_ARCH_TOOLCHAIN_GCC),yy)
+CSRCS += gcov.c
```

Comment:
coverage.c

Reply:
This cannot have the same name as coverage.c in compiler-rt. When compiling using makefile, it will search for coverage.c in the compiler-rt directory, causing it to not find the correct symbol when linking.

Reply:
let's use the full path